src/webapp/components/section/Section.tsx

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import styled from "styled-components";
+
+import i18n from "../../../utils/i18n";
+import { Separator } from "../separator/Separator";
+
+interface SectionProps {
+    title?: string;
+    lastUpdated?: string;
+    children: React.ReactNode;
+    headerButtom?: React.ReactNode;
+    hasSeparator?: boolean;
+    titleVariant?: "primary" | "secondary";
+}
+
+export const Section: React.FC<SectionProps> = React.memo(
+    ({
+        title = "",
+        lastUpdated = "",
+        headerButtom,
+        hasSeparator = false,
+        children,
+        titleVariant = "primary",
+    }) => {
+        return (
+            <SectionContainer $hasSeparator={hasSeparator}>
+                {hasSeparator && <Separator />}
+                <Header>
+                    <TitleContainer>
+                        {title ? <Title $titleVariant={titleVariant}>{title}</Title> : null}
+                        {lastUpdated ? (
+                            <LastUpdatedContainer>
+                                <LastUpdatedTitle>{i18n.t("Last updated: ")}</LastUpdatedTitle>
```

Comment:
`i18next-scanner` fire an error due to `:` separator (nsSeparator)

```suggestion
                                <LastUpdatedTitle>{i18n.t("Last updated: ", { nsSeparator: false })}</LastUpdatedTitle>
```

Reply:
thanks! fixed :tada: